deployments.json

```diff
@@ -124,17 +124,105 @@
+      "address": "0x16d02Dc67EB237C387023339356b25d1D54b0922"
+    },
+    "stkTruToken": {
+      "txHash": "0x2244a3f8f86d5521a1af7b4c871de90228f0c97c5829b5533ed33251fbbeed06",
+      "address": "0xA367647cfc0525CBbdEe6EA036617E0884e3128b"
+    },
+    "stkTruToken_proxy": {
+      "txHash": "0x2fa7c241f85bca05d23b567d8465850c4dd860ffcf88ea2fe1139ba0bae7d04f",
+      "address": "0x23696914Ca9737466D8553a2d619948f548Ee424"
+    },
+    "trueFiPool": {
+      "txHash": "0xed8cc8cf62711922e7bded22887dc3553e27700e4651758fe758ddf698cc1b2f",
+      "address": "0xB3C6fd9a58329172D043C987aBfcE211E9985613"
     },
     "trueFiPool_proxy": {
       "txHash": "0xe2420b341397151bf2790d3b7e7c77f511830c882393ed81d8b4ffcd9f8f6541",
       "address": "0xa1e72267084192Db7387c8CC1328fadE470e4149"
+    },
+    "truPriceOracle": {
+      "txHash": "0xe47889c82379edccc04f3b2770d3318e2c0db7e4b5ac75c183bd6de2743df93d",
+      "address": "0xF7d7edec33A36A4d72EB1262D21d9A4444e4Ea74"
+    },
+    "loanFactory": {
+      "txHash": "0xb4ef32fce3d4331e614e990c7fa253feec321fee7d1fcf7977fe919518052afc",
+      "address": "0xC20500Df4A76B671f5166f6A0E4f36A8F5CFC177"
+    },
+    "loanFactory_proxy": {
+      "txHash": "0x40eb8b4627c9335138ef57c1fca17fed1dd4455dcc6cdb117ba81a065d1e7955",
+      "address": "0x4ACE6dE67E9a9EDFf5c2d0a584390Fb5394119e7"
+    },
+    "liquidator": {
+      "txHash": "0xae18e9d9cbd359448fa8da885b505920e2cf5bf38a2ae722e16755fc80b80d43",
+      "address": "0xA5C6B8930373972c5b67cd8bF4F3DaDBDA82F772"
+    },
+    "liquidator_proxy": {
+      "txHash": "0xbbc189b5cb149d528df85e1574703290096e906025bfd17ac4a00bd8decfee18",
+      "address": "0x76dd4921C99AC6b61b3a98f9fa6f181cA6D70c77"
+    },
+    "stkTruToken_LinearTrueDistributor": {
+      "txHash": "0x73def730be468e897632a508d0a6254fbe3f1407f92b138ff0d3b5312c65fe56",
+      "address": "0x1Bd6423320f8450A4bcD64E16F9cc228f589d1B9"
+    },
+    "stkTruToken_LinearTrueDistributor_proxy": {
+      "txHash": "0xcc12ea7b72ae8c0be09bb5b5830e7e73ba326f2f3f9280a019b033422631c576",
+      "address": "0xecfD4F2C07EABdb7b592308732B59713728A957F"
+    },
+    "trueRatingAgencyV2": {
+      "txHash": "0x45c0df53677d2f1a63686c66ebe893c6f0c2e40afb5c92e09d7db76d006a52b7",
+      "address": "0x353488058bA40b280B73C06FEAdA42CA4d61f7Fc"
+    },
+    "trueRatingAgencyV2_proxy": {
+      "txHash": "0x1e3e717beac53ffa877fed6bcfe1dc0a756b61870be2d857722cd360fd9bd4ca",
+      "address": "0x05461334340568075bE35438b221A3a0D261Fb6b"
+    },
+    "ratingAgencyV2Distributor": {
+      "txHash": "0xa95a7a398b323c81e9027235e2893537c42cdfbf7769e73f8af59386d556e813",
+      "address": "0x87d1616B9B3a0fD756EF4B4Abff29B30ab813f42"
+    },
+    "ratingAgencyV2Distributor_proxy": {
+      "txHash": "0xbc3ebd3bc5ee273dd84811e77c067e629d786effa78099303f71998bd986ba70",
+      "address": "0x6151570934470214592AA051c28805cF4744BCA7"
+    },
+    "timelock": {
```

Comment:
remove timelock, timelock_proxy, govornorAlpha, and govornorAlpha_proxy.

these contracts are not going to be used, we can add them when we deploy new contracts

Reply:
This PR just updates deployments.json to match the current spreadsheet, for any addresses that would be touched by `yarn deploy:truefi`. At the moment this is mostly meant to cut down on areas where we're publishing two statements that say different things, even though they should be the same. And it lets us confirm (in a dry run) which contracts will be updated if we were to run the script.

If we decide to not use these contract addresses in the future, then we can always delete them at that time. But I'm wary of introducing data discrepancies *now* based on *future* actions we plan to take. I think the right time to delete these addresses should be when we deploy new ones to replace them.

That all said, I'm not strongly attached to keeping these addresses, if you insist.

Reply:
that's fine